docs/cli-reference.md

```
@@ -137,8 +201,15 @@ List all instances in the current tenant.
 ```bash
 cloud instance list
 cloud instance list --json  # JSON output
+cloud instance list --limit 10 --offset 0  # Pagination
 ```
 
+**Flags**:
+| Flag | Default | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)